test/library.cpp

```diff
@@ -267,6 +267,44 @@ const char * sampleOpdsStream = R"(
 
 )";
 
+#ifdef _WIN32
+const char sampleLibraryXML[] = R"(
+<library version="1.0">
+  <book
+        id="raycharles"
+        path=".\zimfile.zim"
+        url="https://github.com/kiwix/libkiwix/raw/master/test/data/zimfile.zim"
+        title="Ray Charles"
+        description="Wikipedia articles about Ray Charles"
+        language="eng,spa"
+        creator="Wikipedia"
+        publisher="Kiwix"
+        date="2020-03-31"
+        name="wikipedia_en_ray_charles"
+        tags="wikipedia;_category:wikipedia;_pictures:no"
+        articleCount="284"
+        mediaCount="2"
+        size="556"
+      ></book>
+  <book
+        id="example"
+        path=".\example.zim"
+        title="An example ZIM archive"
+        description="An eXaMpLe book added to the catalog via XML"
+        language="deu"
+        creator="Wikibooks"
+        publisher="Kiwix & Some Enthusiasts"
+        date="2021-04-11"
+        name="wikibooks.de"
+        tags="unittest;wikibooks;_category:wikibooks"
+        articleCount="12"
+        mediaCount="0"
+        size="126"
+      ></book>
+</library>
+)";
+#define LIBRARY_PATH ".\\test\\library.xml"
+#else
```

Comment:
> Too much code duplication.

Done in a fixup commit

> BTW, what about making libkiwix accept unix-style path separators on all platforms?

I have thought about this too. It may even be necessary if user share it `library.xml` between OS. But this has to be done as an improvement.